front-end/src/components/cookie/cookie-view.module.scss

```diff
@@ -0,0 +1,105 @@
+/*** 
+    Cookie view uses Bootstrap modal to display the view. As such, it uses some style overrides 
+    that come from the Bootstrap css.
+***/
+
+@import "src/assets/stylesheets/fonts";
+@import "src/assets/stylesheets/variables";
+@import "src/assets/stylesheets/globals";
+
+// Cookie view related constants
+$cookie-view-font-size: 1.4rem;
+$cookie-body-letter-spacing: -0.05rem;
+$cookie-close-button-font-size: 3.2rem;
+
+$color-cookie-view-bg: rgba(228, 239, 248, 1);
+
+.cookie-modal-backdrop {
```

Comment:
Isnt it better to compose class name instead of extending styles from a class?
At least i feel like its more readable when you try to scan over the main script file (react file) with descriptive class names like
`.cookie-button .active` than `.cookie-button-active`

Reply:
You are absolutely right. It is better, and much more readable to compose class names instead of extending styles. This scenario is a special case however, primarily because of the controls that react-bootstrap provides us. It provides us with props to specify custom classnames to the dialog and the backdrop. However, it doesn't allow us any way to add a custom class to the root dom of the modal. Because of that, the position rules need to be added to the backdrop and dialog custom classes separately, which is why I grouped the position rules into one class and extended them. What you suggested will and should be followed in our own defined components.

Reply:
You are absolutely right. It is better, and much more readable to compose class names instead of extending styles. This scenario is a special case however, primarily because of the controls that react-bootstrap provides us. It provides us with props to specify custom classnames to the dialog and the backdrop. However, it doesn't allow us any way to add a custom class to the root dom of the modal. Because of that, the position rules need to be added to the backdrop and dialog custom classes separately, which is why I grouped the position rules into one class and extended them. What you suggested will and should be followed in our own defined components.

Reply:
yeah then its alright

Reply:
though, perhaps if this is ""special case"" there should be a code comment to explain